### ISSUE TITLE
Weird behavior with Dynamic Virtual Desktops

### ISSUE THREAD

**Issue by Solarinas** [tcorreabr/Parachute#13]
Hey I absolutely love this kwin script you made. Legit made my day when I saw it in the KDE store. Thank you so much for your work

I've been experiencing a weird issue with another kwin script I use called Dynamic Virtual Desktops. It seems like every time I create or remove a virtual desktop with this add-on enabled it will automatically move to the first workspace

Totally understandable if this is super low priority for you since it's a different kwin script. Just thought it would be good to document it. Dynamically creating virtual desktops is a really useful feature for me and having both would be great

**Comment by tcorreabr** [comment]
I also think it would be really cool for Parachute to work well with Dynamic Virtual Desktops. And so I am going to mark this issue as a bug. But at the moment, as you may have guessed, there is still a lot to be done for Parachute to work satisfactorily in isolation. So I'll focus on that for now.

**Comment by d86leader** [comment]
This happens not just with dynamic workspaces scrips. If you manually add or remove a desktop (eg by using a button in a pager right-click menu), you will be moved to the first desktop.

**Comment by fitrh** [comment]
Yeah, this bug not just happen with dynamic workspace scripts, but also with [Virtual Desktop Bar](https://github.com/wsdfhjxc/virtual-desktop-barl) pasmoid, which have dynamic workspace feature

**Comment by iuriSpinassi** [comment]
I'm looking forward to this release @tcorreabr! I had to create lots of virtual desktops to use this feature.
